css-layout-classes/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header></header>
+    <main>
+      <div class="container">
+        <div class="row">
+          <div class="column-full">
+            <h1>Choose your Character!</h1>
+          </div>
+        </div>
+        <div class="row">
+          <div class="column-two-third">
+            <div class="column-one-second"><img src="images/link.png" /></div>
+            <div class="column-one-second">
+              <h2>Super Smash Bros. Ultimate</h2>
+              <p>
+                Super Smash Bros. Ultimate received universal acclaim from
+                both......
+              </p>
+            </div>
+          </div>
+          <div class="column-one-third">
+            <img src="images/yoshi.png" />
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="column-one-fourth"><img src="images/kirby.png" /></div>
+          <div class="column-one-fourth"><img src="images/fox.png" /></div>
+          <div class="column-one-fourth"><img src="images/pikachu.png" /></div>
+          <div class="column-one-fourth"><img src="images/ness.png" /></div>
+        </div>
+
+        <div class="row">
+          <div class="column-one-second">
+            <div><h2>Donkey Kong</h2></div>
+            <div>
+              <p>
+                Donkey Kong is a serious of video games featuring the adventures
+                of a gorilla called Donkey Kong......
+              </p>
+            </div>
+          </div>
+          <div class="column-one-second">
+            <img src="images/dk.png" />
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="column-one-second">
+            <img src="images/mario.png" />
+          </div>
+          <div class="column-one-second">
+            <div><h2>Mario</h2></div>
+            <div>
+              <p>
+                Mario is a fictional character in the Mario video game
+                franchise, owned by Nintendo and created by Japanese......
+              </p>
+            </div>
+          </div>
```

Comment:
These columns are different sizes. The one with the text should be taking up 2/3rds whereas the image will be the one-third